src/Libraries/PythonNodeModelsWpf/IronPythonCompletionProvider.cs

```diff
@@ -205,7 +205,7 @@ public IronPythonCompletionProvider(string dynamoCoreDir)
             // Determine if the Python Standard Library is available in the given context
             if(!String.IsNullOrEmpty(dynamoCoreDir))
             {
-                pythonLibDir = dynamoCoreDir + @"\IronPython.StdLib.2.7.8";
+                pythonLibDir = Path.Combine(dynamoCoreDir + @"\IronPython.StdLib.2.7.8");
```

Comment:
@alfarok this is not right - it should be: `Path.Combine(dynamoCoreDir, "IronPython.StdLib.2.7.8");`

Reply:
@mjkkirschner nice catch, fixing now